src/components/Home/ListOfGames/index.tsx

```diff
@@ -10,42 +9,91 @@ import {
   Title,
 } from "./styles";
 
-const ListOfGames = () => {
-  const url = "https://public.connectnow.org.uk/applicant-test/";
```

Comment:
No need to store it inside the component. At best keep it outside a component's lifecycle if you can.